docs/user_guide/assessment.rst

```diff
@@ -1,21 +1,109 @@
+what ways?
+
+The steps of the assesment are as follows:
+1. Identify harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
 
-The :py:mod:`fairlearn.metrics` module provides the means to assess fairness-related
-metrics for models. This applies for any kind of model that users may already
-use, but also for models created with mitigation techniques from the
-:ref:`mitigation` section. The :ref:`dashboard` provides a visual way to
-compare metrics between models as well as compare metrics for different groups
-on a single model.
+We next examine these four steps in more detail.
 
-Ungrouped Metrics
+1. Identify harms
 ^^^^^^^^^^^^^^^^^
 
+For example, in a system for screening job applications, qualified candidates 
+that are automatically rejected experience an allocation harm. In a 
+speech-to-text transcription system, high error rates constitute harm in the 
+quality of service.
+
+2. Identify the groups that might be harmed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular application. For example, for 
+speech-to-text transcription, groups based on the regional dialect or being a  
+native or a non-native speaker.
+
+It is also important to consider group intersections, for example, in addition
+to considering groups according to gender and groups according to race, it is 
+also important to consider their intersections (e.g., Black women, Latinx 
+nonbinary people, etc.).
+
+3. Quantify harms
+^^^^^^^^^^^^^^^^^
+
+Define metrics that quantify harms or benefits:
+
+* In job screening scenario, we need to quantify the number of candidates that are classified as "negative" (not recommended for the job), but whose true label is "positive" (they are qualified). One possible metric is the false negative rate: fraction of qualified candidates that are screened out.
+
+* In speech-to-text scenario, the harm could be measured by word error rate, number of mistakes in a transcript divided by the overall number of words.
+
+4. Compare quantified harms across the groups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The workhorse of fairness assessment are disaggregated metrics, which are 
+metrics evaluated on slices of data. For example, to measure harms due to 
+errors, we would begin by evaluating the errors on each slice of the data that 
+corresponds to a group. If some of the groups are seeing much larger errors 
+than other groups, we would flag this as a fairness harm.
+
+To summarize the disparities in errors (or other metrics), we may want to 
+report quantities such as the difference or ratio of the metric values between 
+the best and the worst slice. In settings where the goal is to guarantee 
+certain minimum quality of service (such as speech recognition), it is also 
```

Comment:
Slight nitpick: if there's a minimal QoS goal, then surely 'minimum across all groups' _is_ the key fairness metric?

Reply:
Added "across all groups" to this sentence (so now reads "minimum quality of service _across all groups_"), please let me know if this is sufficient